extensions/interactions/CodeRepl/CodeRepl.html

```diff
@@ -52,15 +52,19 @@
 <script type="text/ng-template" id="interaction/CodeRepl">
   <div class="code-repl-container">
     <div>
-      <div class="code-repl-input-box">
-        <div ui-codemirror="{onLoad: initCodeEditor}">
+      <!-- This ng-form is needed for submitting answer on hitting enter. -->
+      <form ng-submit="runAndSubmitCode(code)">
```

Comment:
Just FYI, a form probably isn't needed here, since hitting enter shouldn't submit the code. I'm fine either way; just wanted to let you know.